ui/imports/shared/controls/AmountInput.qml

```diff
@@ -32,6 +32,21 @@ Input {
         id: d
 
         property real amount: 0
+
+        readonly property var nonDigitCharacterRegExp: {
+            const localizedNumbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9].map(
```

Comment:
This is a JS RegExp that understands Unicode, so this works fine (note the `u` modifier):
```
readonly property var nonDigitCharacterRegExp = /[^\d]/gu
```

Reply:
OK, fine then :) I just thought this is a bit overkill and slow

Reply:
For this purpose even being slow, it doesn't affect performance at all. I was looking for sth easier but finally landed with that :)